tests/test_assets.py

```diff
@@ -308,28 +307,20 @@ def test_lookup_symbol_fuzzy(self):
             finder.retrieve_asset(i) for i in range(3)
         )
 
-        for i in range(2):  # we do it twice to test for caching bugs
+        # we do it twice to catch caching bugs
+        for i in range(2):
+            # Shouldn't find this with no fuzzy_str passed.
```

Comment:
I'm confused what this comment means- lookup symbol doesn't take a fuzzy str. Is what you meant rather that we shouldn't find either of the first two cases since the symbol string doesn't contain a fuzzy character?